src/Todos.jsx

```diff
@@ -46,24 +46,26 @@ function Todos() {
         );
         const fetchedData = await response.json();
         const newData = fetchedData.map((toDo) => ({ ...toDo, id: v4() }));
-        setToDoList(newData);
+        const newDataWithShow = newData.map((toDo) => {
+            return { ...toDo, show: true };
+        });
+        setToDoList(newDataWithShow);
     }, []);
     useEffect(() => {
         fetchData();
     }, [fetchData]);
     const filterToDoList = useCallback(
         (searchTermProp) => {
-            const combinedList = [...toDoList, ...toDoListfliteredOut];
-            const filteredIn = combinedList.filter((toDo) =>
-                toDo.task.includes(searchTermProp),
-            );
-            const filteredOut = combinedList.filter(
-                (toDo) => !toDo.task.includes(searchTermProp),
+            setToDoList(
+                toDoList.map((toDo) => {
+                    if (toDo.task.includes(searchTermProp)) {
```

Comment:
great. now instead of `String.prototype.includes` i want you to use regex that will ignore case sensitivity (f.e "abc" and "ABC" should return the same results)